### ISSUE TITLE
Create Ruby script for file generation

### ISSUE THREAD

**Issue by woodbe** [itc-wgtools/cPP-Tools#40]
This is probably very simple, but I want to create 2 scripts that we would use to consistently generate the proper files. 

Basically what I'm thinking is to provide a script that would automatically apply the settings here:

https://github.com/itc-wgtools/cPP-Tools/wiki/Asciidoctor-output-generation-notes

This way all someone would need to do is to call say "ad-html <filename>" or "ad-pdf <filename>" and it would automatically apply the settings. Obviously someone could modify this, but this would provide the default.

**Comment by woodbe** [comment]
@atsecsysad Do you know anything about creating Ruby Gems? I was thinking about creating a Gem package to make it easier to have people get the needed scripts, but I'm having trouble getting them to register into the path (I can't figure out the Ruby path system since it doesn't fit what I expect as I don't see paths for some things I can run from anywhere while other things can't).

**Comment by woodbe** [comment]
I have created the initial set as a ZIP file holding the script file. This is fine and will work. Creating the gem though would make it about to be installed easily and also let it be run from anywhere (I could easily make the adjustments to the script now that I have it written and commented). Having it as a gem would also let us publish updates and just tell people to run the update process for it to automatically be downloaded. This would I think make a better ongoing experience, but I can't figure out how to create the Gem such that the program is added to the path so it can be loaded anywhere.

If there are any ideas about this (or someone to ask specifically), please speak up.

**Comment by woodbe** [comment]
I am closing this item. I figured out how to create the package so I can run it from anywhere with the installation of the Gem. I have placed this into a new repository here: https://github.com/itc-wgtools/itc_gentools.

It may be possible to make this a RubyGem later and have it be able to be installed from within Ruby directly, but for the moment the user can download the Gem file and install it manually.
